vng/reader.go

```diff
@@ -11,48 +11,48 @@ import (
 
 // Reader implements zio.Reader for a VNG object.
 type Reader struct {
-	root    *vector.Int64Reader
-	readers []typedReader
+	Root    *vector.Int64Reader
+	Readers []TypedReader
 	builder zcode.Builder
 	val     zed.Value
 }
 
-type typedReader struct {
-	typ    zed.Type
-	reader vector.Reader
+type TypedReader struct {
+	Typ    zed.Type
```

Comment:
We say `typ` because `type` is a Go keyword, but it's ugly.
```suggestion
	Type    zed.Type
```